tests/DevApps/B2CWebAppCallsWebApi/Client/Startup.cs

```diff
@@ -34,7 +34,7 @@ public void ConfigureServices(IServiceCollection services)
                 // This lambda determines whether user consent for non-essential cookies is needed for a given request.
                 options.CheckConsentNeeded = context => true;
                 options.MinimumSameSitePolicy = SameSiteMode.Unspecified;
-                // Handling SameSite cookie according to https://docs.microsoft.com/en-us/aspnet/core/security/samesite?view=aspnetcore-3.1
+                // Handling SameSite cookie according to https://learn.microsoft.com/aspnet/core/security/samesite
```

Comment:
```suggestion
                // Handles SameSite cookies according to the ASP.NET Core documentation at https://learn.microsoft.com/aspnet/core/security/samesite.
```

Reply:
There are several of these, so I'll do a find-'n-replace on my branch to get this phrasing change all in one go.

Reply:
... and I won't forget the period on this one.